.github/workflows/pkgci_regression_test.yml

```diff
@@ -126,245 +127,254 @@ jobs:
           name: ${{ matrix.config-file }}
           path: ${{ env.CONFIG_FILE_PATH }}
 
-  test_models:
-    name: "test_models :: ${{ matrix.name }}"
-    runs-on: ${{ matrix.runs-on }}
-    strategy:
-      fail-fast: false
+  # Disabled while runners are unstable.
+  # test_models:
+  #   name: "test_models :: ${{ matrix.name }}"
+  #   runs-on: ${{ matrix.runs-on }}
+  #   strategy:
+  #     fail-fast: false
 
-      # Note: these jobs should use persistent runners with local caches.
-      # Downloading test files (50GB+) without a cache can take 20+ minutes.
```

Comment:
Unfortunate that all the model tests are disabled here. If the persistent runners are unstable for long, we could enable a selection of tests at least on CPU ephemeral runners at postsubmit time, downloading with no cache support.